<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QTTester</class>
 <widget class="QWidget" name="QTTester">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>554</width>
    <height>532</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QTTester</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="issuesLabel">
         <property name="text">
          <string>Issues</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="issuesList"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="descLabel">
         <property name="text">
          <string>Description</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="descEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="removeButton">
         <property name="text">
          <string>REMOVE</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="addButton">
         <property name="text">
          <string>ADD</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
